sample/SampleHost/Program.cs

```diff
@@ -20,6 +23,23 @@ static void Main(string[] args)
             }
 
             config.CreateMetadataProvider().DebugDumpGraph(Console.Out);
+            config.Tracing.ConsoleLevel = TraceLevel.Off;
+
+            // Build up a LoggerFactory to log to App Insights, but only if this key exists.
+            string instrumentationKey = Environment.GetEnvironmentVariable("APPINSIGHTS_INSTRUMENTATIONKEY");
+            if (!string.IsNullOrEmpty(instrumentationKey))
+            {
+                // build up log levels for any category
+                LogCategoryFilter filterBuilder = new LogCategoryFilter();
```

Comment:
Since the type name is LogCategoryFilter, calling the local filterBuilder seems like a misnomer. If the type is a filter and not a builder/factory, perhaps you should pass it directly into .AddApplicationInsights below, rather than passing in filterBuilder.Filter?